views/player_options.erb

```diff
@@ -22,6 +21,6 @@
         <% end %>
       </ul>
       <button type='submit' class='btn btn-primary'>Play</button>
-    </form>
+      </form>
```

Comment:
I think the indentation is off here because the `<h1>` tag is not closed properly.